### PR TITLE
ENT-3761: Added course skills in the search index

### PR DESCRIPTION
**Pull request by irfanuddinahmad**
Description: This PR adds the skills associated with a course in the search index.

JIRA: https://openedx.atlassian.net/browse/ENT-3761

### CODE REVIEW

**Review by mikix — APPROVED** [review]

**Comment by mikix** [comment]
Is this urgent? If possible, I'd like to avoid landing any changes to our elasticsearch code right now, as we are going to be testing an upgrade to v7 today.

**Comment by irfanuddinahmad** [comment]
> Is this urgent? If possible, I'd like to avoid landing any changes to our elasticsearch code right now, as we are going to be testing an upgrade to v7 today.

No urgency. Thanks for the review. I will not merge until after the upgrade.

File: course_discovery/apps/course_metadata/search_indexes.py [hunk 1]
```diff
@@ -159,6 +160,7 @@ class CourseIndex(BaseCourseIndex, indexes.Indexable):
     prerequisites = indexes.MultiValueField(faceted=True)
     languages = indexes.MultiValueField()
     seat_types = indexes.MultiValueField()
+    skill_names = indexes.MultiValueField()
```
Comment (saleem-latif) [inline course_discovery/apps/course_metadata/search_indexes.py:163]: what would be better? `skill_names` or `skills`? just thinking about the user's perspective when writing query filter.

**Review by saleem-latif — APPROVED** [review]
Just one question otherwise looks good 👍  
Does doing this enables the filtering via query filter on `search/all` ?

**Review by saleem-latif — APPROVED** [review]
The branch needs a rebase with the master.

File: course_discovery/apps/course_metadata/search_indexes.py (hunk 1 above)
Comment (irfanuddinahmad) [reply course_discovery/apps/course_metadata/search_indexes.py:163]: need to differentiate these from the skill_ids ...

**Review by muhammad-ammar — APPROVED** [review]

**Pull request merged**